### ISSUE TITLE
login failed

### ISSUE THREAD

**Issue by r4nchy** [rasmuslos/ShelfPlayer#142]
I am using abs v2.13.4 and the v2.4.5 .ipa ios to login, the server is detected and shows the server version but the login page on the app shows "login failed"

in the logs it says `[2024-09-19 19:34:11.070] INFO: [Auth] User "Administrator" logged in from ip 106.***.***.57`

here is my logs

```
Config /config /metadata
[2024-09-19 19:33:46.245] INFO: === Starting Server ===
[2024-09-19 19:33:46.276] INFO: [Server] Init v2.13.4
[2024-09-19 19:33:46.277] INFO: [Server] Node.js Version: v20.17.0
[2024-09-19 19:33:46.277] INFO: [Server] Platform: linux
[2024-09-19 19:33:46.279] INFO: [Server] Arch: arm64
[2024-09-19 19:33:46.298] INFO: [Database] Initializing db at "/config/absdatabase.sqlite"
[2024-09-19 19:33:46.445] INFO: [Database] Db connection was successful
[2024-09-19 19:33:46.983] INFO: [Database] Db initialized with models: user, library, libraryFolder, book, podcast, podcastEpisode, libraryItem, mediaProgress, series, bookSeries, author, bookAuthor, collection, collectionBook, playlist, playlistMediaItem, device, playbackSession, feed, feedEpisode, setting, customMetadataProvider, mediaItemShare
[2024-09-19 19:33:47.213] WARN: Removed 1 sessions that were 3 seconds or less (Database.js:743)
[2024-09-19 19:33:47.226] INFO: [Database] Server upgrade detected from 2.13.3 to 2.13.4
[2024-09-19 19:33:47.276] INFO: [LogManager] Init current daily log filename: 2024-09-19.txt
[2024-09-19 19:33:47.324] INFO: [BackupManager] 0 Backups Found
[2024-09-19 19:33:47.326] INFO: [BackupManager] Auto Backups are disabled
[2024-09-19 19:33:47.402] INFO: [Watcher] Initializing watcher for "Books".
[2024-09-19 19:33:47.408] INFO: [Watcher] Initializing watcher for "Comik".
[2024-09-19 19:33:47.448] INFO: Listening on port :80
[2024-09-19 19:33:47.939] INFO: [Watcher] "Books" Ready
[2024-09-19 19:33:50.823] INFO: [SocketAuthority] Socket Connected ETaNpADA9q0v3wJCAAAB
[2024-09-19 19:33:52.861] INFO: [Watcher] "Comik" Ready
[2024-09-19 19:33:54.779] INFO: [SocketAuthority] Socket Connected Ym-GqS1ld-t8Hf-dAAAD
[2024-09-19 19:33:59.891] INFO: [SocketAuthority] Socket Connected wEthtHK_lGqcT_A1AAAF

[2024-09-19 19:34:11.070] INFO: [Auth] User "Administrator" logged in from ip 106.***.***.57
[2024-09-19 19:34:30.222] INFO: [Auth] User "Administrator" logged in from ip 106.***.***.57
[2024-09-19 19:34:45.753] INFO: [Auth] User "Administrator" logged in from ip 122.***.***.39
[2024-09-19 19:34:53.782] INFO: [Auth] User "Administrator" logged in from ip 122.***.***.39
[2024-09-19 19:35:32.563] INFO: [Auth] User "Administrator" logged in from ip 122.***.***.39
```

**Comment by rasmuslos** [comment]
Most likely related to #135 … Can you install the app using Xcode and paste the console output here?

**Comment by saalaataa** [comment]
hi @rasmuslos again. with latest version of ABS and SP when I logged out and tried to log back in i got error message:
[2024-09-27 18:15:45.713] INFO: [Auth] User "test" logged in from ip x.x.x.x
[2024-09-27 18:15:45.770] ERROR: [MeController] createBookmark invalid title  (MeController.js:203)
[2024-09-27 18:15:47.295] ERROR: [MeController] createBookmark invalid title  (MeController.js:203)
[2024-09-27 18:16:17.449] INFO: [Auth] User "shelfplayer" logged in from ip x.x.x.x
[2024-09-27 18:16:17.535] ERROR: [MeController] createBookmark invalid title  (MeController.js:203)

I am using dedicated account for SP and created new test one as well to receive same error. It may be ABS issue, but IDK as after I downgraded to ABS 2.12.2 the error disappears. It looks like ABS made significant changes.
